src/pages/Pokemon/Versions.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./pokemon.css";
+import { useState } from "react";
+
+const Versions = ( {blue, red} ) => {
+  const [Text, setText] = useState(blue)
+
+  return (
+    <div className="Test">
+      <div>
+        <p className="blue-p">{Text}</p>
+        <p className="red-p">{setText}</p>
```

Comment:
we don't wanna call `setText` this way mate cos I think this way it's gonna fire every render if i'm not mistaken